src/main/java/seedu/manager/parser/Parser.java

```diff
@@ -930,15 +926,21 @@ private boolean toMarkItem(String status) throws InvalidCommandException, IOExce
      * @throws InvalidCommandException if the command is missing required parts or has an invalid format
      */
     private Command parseCopyCommand(String input, String[] commandParts) throws InvalidCommandException {
```

Comment:
maybe can add the check for duplicate flags here as well and make sure they don't use the '>'  more than once?